src/App.tsx

```diff
@@ -15,48 +17,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  sortAsc = 'abc',
+  sortByLength = 'byLength',
+  initialVaue = '',
+}
```

Comment:
https://aykhanhuseyn.medium.com/typescript-naming-conventions-crafting-maintainable-code-7d872234fe17, read the part about enum naming and also there is an error initialVaue -> initialValue